modules/eks/submodules/k8s/variables.tf

```diff
@@ -1,7 +1,7 @@
 variable "calico_version" {
   type        = string
   description = "Calico operator version."
-  default     = "v3.25.2"
```

Comment:
Wasn't totally sure if this was ok. It's not necessary for this change, but I figured while I'm updating things.

Reply:
only bumped the patch for the same reason, but I am not aware of any issues.